zulipterminal/ui_tools/boxes.py

```diff
@@ -40,6 +40,8 @@ def __init__(self, view: Any) -> None:
         self.stream_id = None  # type: Optional[int]
         self.recipient_user_ids = []  # type: List[int]
         self.msg_body_edit_enabled = True
+        self.in_edit_message_box = False
```

Comment:
This extra boolean can be avoided by making use of `stream_box_edit_view`, because it seems to indicate a message being edited.